repos/system_upgrade/el7toel8/actors/authselectscanner/libraries/library.py

```diff
@@ -0,0 +1,268 @@
+import ConfigParser
+import os
+import re
+import StringIO
+import textwrap
+
+from leapp.libraries.stdlib import run, CalledProcessError
+from leapp.models import Authselect
+
+
+def read_file(config):
+    """
+        Read file contents. Return empty string if the file does not exist.
+    """
+    if not os.path.isfile(config):
+        return ""
+    with open(config) as f:
+        return f.read()
+
+
+def is_service_enabled(service):
+    """
+        Return true if @service is enabled with systemd, false otherwise.
+    """
+    try:
+        run(["/usr/bin/systemctl", "is-enabled", "{}.service".format(service)])
+    except (OSError, CalledProcessError):
+        return False
+
+    return True
+
+
+class ConfigFile(object):
+    """
+        Base class for config parsers.
+    """
+
+    def __init__(self, content):
+        self.config = ConfigParser.ConfigParser()
+        self.config.readfp(StringIO.StringIO(textwrap.dedent(content)))
+
+    def get_string(self, section, option):
+        if not self.config.has_option(section, option):
+            return None
+
+        str = self.config.get(section, option)
```

Comment:
str is a builtin for the string type, this is a dangerous name - considering you don't do anything else than stripping just do this:
```suggestion
        return self.config.get(section, option).strip('"\'')
```